.gitlab-ci.yml

```diff
@@ -127,6 +127,8 @@ test-r-release-vanilla: # check minimal installation, no suggested deps, no vign
   dependencies:
   - mirror-packages
   - build
+  variables:
+    TEST_DATA_TABLE_MEMTEST: "TRUE"
```

Comment:
this activates memory stats collection, it is now active for fastest test job, from 3min it is now 14min.